scripts/python/lm_eval.py

```diff
@@ -0,0 +1,208 @@
+from bcipy.language.model.kenlm import KenLMLanguageModel
+from bcipy.language.model.mixture import MixtureLanguageModel
+from bcipy.language.model.unigram import UnigramLanguageModel
+from bcipy.language.model.causal import CausalLanguageModel
+from bcipy.language.main import ResponseType
+from math import log10
+from timeit import default_timer as timer
+from bcipy.language.main import BACKSPACE_CHAR, SPACE_CHAR, alphabet
+import argparse
+import numpy as np
+import sys
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--verbose', dest='verbose', type=int, required=True,
+        help='0: Only output model averages\n1: Output results from each phrase\n2: Output results from each character')
+
+    parser.add_argument('--model', dest='model', type=int, required=True,
+        help='1: Unigram\n2: Mixture (80/20 Causal GPT-2/Unigram)\n3: KenLM n-gram\n4: Causal Hugging Face')
+
+    parser.add_argument('--phrases', dest='phrases', type=str, required=True,
```

Comment:
This is a great script to have. I'm opinionated about scripts living here in the codebase, though. Often, it's where untested (as in unittests and linting!) and feature-rich code gets rooted and stays. It gets updated less reliably and goes stale even if it becomes a critical part of our workflow. I have been bitten several times and hope desperately to deprecate this in the repo. 

I'd be happy to keep it in here for now if we make another ticket to pull out what we can into the repo as supported code (and then use a demo to evoke it!)

Reply:
Ticket created! 
https://www.pivotaltracker.com/story/show/184794336